### PR TITLE
Result tab

### PR DESCRIPTION
**Pull request by reshetniktati**

### CODE REVIEW

File: src/app/group/tabs/result-tab/result-tab.component.ts [hunk 1]
```diff
@@ -20,18 +20,18 @@ export class ResultTabComponent implements OnInit {
...
+  // currentUserId: string = '1993036'; //admin
+  currentUserId: string = '128736';//student
 
 
   constructor(private userService: UserService, private route: ActivatedRoute) { }
 
   ngOnInit() {
-  this.userService.getCuratorByGroupId(this.currentGroupId)
-    .subscribe(curator => this.curator = curator);
+    this.userService.getCuratorByGroupId(this.currentGroupId)
```
Comment (IamMikl) [inline src/app/group/tabs/result-tab/result-tab.component.ts]: unsubscribe on destroy?

File: src/app/group/tabs/result-tab/result-tab.component.ts [hunk 2]
```diff
@@ -20,18 +20,18 @@ export class ResultTabComponent implements OnInit {
...
+      .subscribe(curator => this.curator = curator);
 
-  this.userService.getUserById(this.currentUserId)
-  .subscribe(user => this.currentUser = user);
+    this.userService.getUserById(this.currentUserId)
+    .subscribe(user => this.currentUser = user);
```
Comment (IamMikl) [inline src/app/group/tabs/result-tab/result-tab.component.ts]: unsubscribe on destroy?

File: src/app/group/tabs/result-tab/result-tab.component.ts (hunk 1 above)
Comment (reshetniktati) [reply src/app/group/tabs/result-tab/result-tab.component.ts]: fixed

File: src/app/group/tabs/result-tab/result-tab.component.ts (hunk 2 above)
Comment (reshetniktati) [reply src/app/group/tabs/result-tab/result-tab.component.ts]: fixed

**Pull request merged**